Fix test test_numpy_torch_operators as the operator matrixmultiply
issues a ValueError when called on two scalars.

Author: Jakob Schiotz, Tech. Univ. Denmark
Email: schiotz@fysik.dtu.dk
Date: 8. November 2023

diff --git a/test/dynamo/test_misc.py b/test/dynamo/test_misc.py
index c5ed05a..5c6fbcb 100644
--- a/test/dynamo/test_misc.py
+++ b/test/dynamo/test_misc.py
@@ -1324,7 +1324,7 @@ class MiscTests(torch._dynamo.test_case.TestCase):
             try:
                 # TODO try a bit harder
                 result = op(t1, t2)
-            except (RuntimeError, TypeError, IndexError):
+            except (RuntimeError, TypeError, IndexError, ValueError):
                 continue
             cnts = torch._dynamo.testing.CompileCounter()
             opt_fn = torch._dynamo.optimize(cnts)(fn)
